tests/functional/test_ems_end2end.py

```diff
@@ -600,11 +612,36 @@ def validate_test_server(cls):
         cls.assert_test(lambda: resp.json.get("configuration") == WEAVER_CONFIGURATION_EMS,
                         message="weaver must be configured as EMS.")
 
+    @pytest.mark.skip(reason="Workflow not working anymore. IO to be repaired.")
```

Comment:
avoid globally skipping test
anywhere that do run it will not do so anymore with this

if you need to run another subset of tests excluding this one, add more tags to select the specific tests you want

Reply:
ok, changed to `xfail` then, since it don't currently work